.github/workflows/publish.yml

```diff
@@ -0,0 +1,53 @@
+name: Publish Builds
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        target: [release-chrome, release-firefox, beta-firefox]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 16
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      - name: Dependencies
+        run: |
+          npm ci
+          npm i -g web-ext
+      - name: Build
+        run: |
+          npm run generate-etags
```

Comment:
I wonder if these three calls should be a script itself? Or perhaps remove the package.json duplication if it's never going to be manual?